### DISCUSSION TITLE
Deluge Support

### DISCUSSION THREAD

**Discussion by razaqq** [jesec/flood#217] [Ideas]
Besides rTorrent and Transmission and qBittorrent, Deluge is another very popular torrent client.
The webui isnt exactly outdated, but getting a modern ui like this would be amazing.

Some people seem to still be using deluge 1.3.X, which is not compatible with the 2.X rpc protocol.
I would suggest to focus on 2.X first tho, since sooner or later that will replace the older version.

**Comment by jesec** [comment]
It is on the roadmap. However, Deluge RPC interface is a bit difficult to navigate and it would certainly take some significant time and efforts.

**Comment by razaqq** [comment]
ill see if i get the time to work on this a bit and start a pr
unfortunately not a fan at all of js/ts

**Comment by C84186** [comment]
I'm going to weigh in here with an opinionated, and potentially unhelpful take -

Having used Deluge, what's cool about it is the plugin support, and a few other features. 
I found the performance of Deluge fairly unacceptable at scale, (I think >1000 seeding, give or take? on an AMD threadripper 1920x ).

Plugin support is probably moot when using deluge behind flood, and the performance really isn't great. I guess it would be nice to have it, but I (personally) wouldn't urge @jesec to implement it (that's just my opinion)
